src/script_lua.c

```diff
@@ -238,9 +238,12 @@ static void redisProtocolToLuaType_Error(void *ctx, const char *str, size_t len,
          * to push elements to the stack. On failure, exit with panic. */
         serverPanic("lua stack limit reach when parsing redis.call reply");
     }
-    lua_newtable(lua);
-    lua_pushstring(lua,"err");
-    lua_pushlstring(lua,str,len);
+    sds err_msg = sdscatlen(sdsnew("-"), str, len);
```

Comment:
Make sure `lua_checkstack` is still valid.

Reply:
We are still pushing up to 3 elements into the Lua stack, `lua_checkstack` still holds here.